src/templates/en/splash.html

```diff
@@ -8,28 +8,170 @@
   <link rel="stylesheet" href="/static/css/splash.css">
 {% endblock %}
 
-{% block main %}
-<section>
-  <div>
-    <img src="../static/images/ha.png" alt="HTTP Archive">
+{% block content %}
+<div id="home">
+  <header>
+    <section class="nav">
+      <div class="logo">
+        <b>Web Almanac</b>
+      </div>
+      <nav class="nav-items">
+        <li>
+          <a href="#">Facts and stats</a>
+        </li>
+        <li>
+          <a href="#">Contributors</a>
+        </li>
+        <li>
+          <a href="#">Methodology</a>
+        </li>
+        <li>
+          <a href="#">Explore older reports</a>
+        </li>
+        <select>
+          <option>2019</option>
+        </select>
+      </nav>
+      <div class="explore">
+        <button>
+          Start Exploring
+        </button>
+        <select class="language-switcher">
+            {% if supported_languages is iterable %}
+              {% for l in supported_languages %}
+                <option
+                  {% if l == language %}selected="selected"{% endif %}
+                  value="{{ url_for(request.endpoint, **get_view_args(lang=l.lang_code)) }}">
+                  {{ l }}
+                </option>
+              {% endfor %}
+            {% endif %}
+          </select>
+      </div>
+    </section>
+    <section class="intro">
+      <h1>Web Almanac</h1>
+      <h2>
+        <b>HTTP</b> Archive's annual state of the <b>web report</b>
+      </h2>
+      <div>
+        Our mission is to combine the raw stats and trends of the HTTP Archive with the expertise of the web community. The Web Almanac is a comprehensive report on the state of the web, backed by real data and trusted web experts. It is comprised of 20 chapters spanning aspects of page content, user experience, publishing, and distribution.
+      </div>
+      <button>
+        Start exploring
+      </button>
+    </section>
+  </header>
+  <section class="mobile-web-container">
+    <div class="mobile-web">
+      <h1>Mobile Web</h1>
+      <div>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Enim facilisis gravida neque convallis a cras semper auctor neque.
+      </div>
+      <div class="mobile-web-data">
+        <div class="mobile-web-data-item">
+          <div>36%</div>
+          <div>Lorem ipsum</div>
+        </div>
+        <div class="mobile-web-data-item">
+          <div>120M</div>
+          <div>Lorem ipsum</div>
+        </div>
+        <div class="mobile-web-data-item">
+          <div>07/10</div>
+          <div>Lorem ipsum</div>
+        </div>
+      </div>
+      <button>
+        Read the R2D2C3P0 Chapter
+      </button>
+    </div>
+  </section>
+  <section class="contributors-container">
```

Comment:
See if you can also add the big "91 people" text on the right.

![image](https://user-images.githubusercontent.com/1120896/63056954-29bc2700-beb7-11e9-8cf4-589b0e8d8bb7.png)